tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -49,6 +50,10 @@ class RNTupleModel {
    std::unique_ptr<RFieldRoot> fRootField;
    /// Contains field values corresponding to the created top-level fields
    std::unique_ptr<REntry> fDefaultEntry;
+   /// Keeps track of which field names are taken.
+   std::unordered_set<std::string> fFieldNames;
+
+   void EnsureUniqueFieldName(const std::string& fieldName);
```

Comment:
use `std::string_view`

Reply:
Done in 49624995939ab0ee78c73384c5d65e83b343ce49.